src/commands/createAccountsService.js

```
@@ -0,0 +1,74 @@
+      return false;
+    }
+
+    logger.warning(
+      `No credentials found in Secrets Manager for existing user, creating new ones...`
+    );
+    return true;
+  }
+
+  async displayCredentialInfo(accountId, username) {
+    const secretName = `iam-user/${accountId}/${username}`;
+    const secretArn = `arn:aws:secretsmanager:${config.accountId}:secret:${secretName}`;
```

ESLint

[error] 'config' is not defined.

(no-undef)